spark-etl/src/main/scala/geotrellis/spark/etl/config/json/ConfigFormats.scala

```diff
@@ -128,10 +128,35 @@ trait ConfigFormats {
       }
   }
 
+  implicit val cassandraPartitionStrategyFormat = new JsonFormat[CassandraPartitionStrategy]{
```

Comment:
I don't think Spray JSON has a nice way to automatically encode/decode enums encoded as ADTs.  There may be other add-on libraries out there that know how to automate this boilerplate as a macro or something... not sure what this project's feeling is on that sort of thing.